Restructured readme and merged Physics documentation into one single file

ojha-aditya opened the pull request:
As mentioned in issue #25, relabeling the README as HW description and creating a new README file. The contents have not been written yet for the new README.

ojha-aditya committed:
Create Bell's Inequality

ojha-aditya committed:
Merge branch 'ubsuny:main' into ojha-aditya/CP1-24-HW-workbranch

ojha-aditya committed:
Rename README.md to HW description and Grading Rubric.md

ojha-aditya committed:
Create NEW-README.md

laserlab commented on Bell's Inequality:
```diff
@@ -0,0 +1 @@
+Markdown code describing the Physics of Bell's inequality
```
add from already existing documentation

ojha-aditya replied:
On it. I was actually adding some content to this file, so shall I do that and then create a separate pull request as my second one for this?
And remove this blank file for time being?

laserlab replied:
No this should be done in this PR.

ojha-aditya replied:
You mean adding from already existing documentation should be done in this PR or my own changes as well?

laserlab replied:
Ideally both

ojha-aditya replied:
I have added the documentation from existing 3 files with my own into this.

laserlab commented on HW description and Grading Rubric.md:
Please use CamelCase and not spaces in filenames

ojha-aditya replied:
Done.

laserlab commented on NEW-README.md:
Should definetly not be named NEW-anything, also will only merge with actual content in it in such a late stage of the project.

ojha-aditya replied:
So should we only keep the original readme with some edits at this stage, instead of the new separate files for Grading Rubric and README?

laserlab replied:
No, it means that this is a version control system and so filenames should not indicate a version. NEW clearly does this.

ojha-aditya replied:
Got it. Changed that.

laserlab reviewed (changes requested):
- Filenames
- content missing

ojha-aditya committed:
Merge branch 'ubsuny:main' into ojha-aditya/CP1-24-HW-workbranch

laserlab commented:
add label and milestone

laserlab commented:
fix typo in PR title

ojha-aditya committed:
Rename HW description and Grading Rubric.md to HomeWorkDescriptionAndGradingRubric.md

ojha-aditya committed:
Rename NEW-README.md to README.md

laserlab commented on README.md:
please add content instead of placeholder that late in the project

ojha-aditya replied:
Sure. I will make the changes to both README and the Bell's Inequality files in a bit and commit them in a short while. Are there any particular things to add to README?

laserlab replied:
As I said in #25 it should describe the project and give an overview of the files in the repo

ojha-aditya replied:
Made the suggested changes.

ojha-aditya committed:
Update and rename Bell's Inequality to Bell'sInequalityTheory

ojha-aditya committed:
Update Bell's Inequality - Theory

ojha-aditya committed:
Update Bell's Inequality - Theory

ojha-aditya committed:
Rename Bell's Inequality - Theory to BellsInequalityTheory.md

ojha-aditya committed:
Update BellsInequalityTheory.md

ojha-aditya committed:
Update README.md

Updated Readme for the description of the repository

ojha-aditya committed:
Rename BellInequalityTheory.md to BellsInequalityTheory.md

Merged the three documentation files and made citations format uniform, added the Mathematics of Bell's inequality

ojha-aditya committed:
Update README.md

Added repository contents description to README file
